backend/app/main.py

```
@@ -215,7 +217,45 @@ async def cleanup_old_diagnosis_images():
+#
+# Graceful fallback: MCP 빌드/http_app 실패 시 _review_mcp_app=None 으로 폴백.
+# Bridge·RAG·사진 정리와 동일 패턴 — MCP 만 비활성화되고 REST 본체는 정상 기동.
+# stateless_http=True: 단발 HTTP 호출도 세션 핸드셰이크 없이 동작 (T4 progress SSE 영향 없음).
+_review_mcp_app = None
+try:
+    _review_mcp = build_review_mcp()
+    _review_mcp_app = _review_mcp.http_app(path="/", stateless_http=True)
+except Exception as _mcp_exc:  # noqa: BLE001 — MCP 실패가 BE 기동 막지 않음
+    logging.getLogger(__name__).exception(
+        "review_mcp.build_failed — /mcp 비활성, REST 본체는 정상 기동 (err=%s)",
+        _mcp_exc,
```

Ruff (0.15.12)

[warning] Redundant exception object included in `logging.exception` call

(TRY401)